### ISSUE TITLE
Feature: Expose HTTP Chrome endpoint to provide details of debugging target

### ISSUE THREAD

**Issue by auchenberg** [node-inspector/node-inspector#684]
Chrome and other compliant debugging-servers is in addition to the WebSocket-endpoint, also exposing a HTTP endpoint where it's possible to get a JSON description of the current target.

The interface is as following:

**/json**

```json
{
   "description": "",
   "devtoolsFrontendUrl": "/devtools/devtools.html?ws=localhost:3058/devtools/page/5DC5F978-E0B2-7DB4-8962-7CB0AC46AD1A",
   "id": "5DC5F978-E0B2-7DB4-8962-7CB0AC46AD1A",
   "title": "",
   "type": "page",
   "url": "http://www.google.com/",
   "webSocketDebuggerUrl": "ws://localhost:3058/devtools/page/5DC5F978-E0B2-7DB4-8962-7CB0AC46AD1A"
}
```

Other methods are documented here: https://github.com/buggerjs/bugger-daemon#api

I'm proposing to implement **/json**  and **/json/version** as it would allow tools like Adobe Brackets and Chrome DevTools App to connect to node-inspector server, and automatically "discover" the target details. 

Should be trivial to implement.

**Comment by 3y3** [comment]
I'm :+1 here.

@auchenberg , are you ready to implement it?

**Comment by auchenberg** [comment]
Sure. I can give it a try next week when I'm back to a stable connection :)

**Comment by auchenberg** [comment]
Note: Implementation in Stetho, https://github.com/facebook/stetho/blob/master/stetho/src/main/java/com/facebook/stetho/inspector/ChromeDiscoveryHandler.java#L111

**Comment by 3y3** [comment]
Closed in #689
